colab/static/js/subscriptions.js

```diff
@@ -0,0 +1,32 @@
+$(function (){
+
+  ENTER_BUTTON = 13;
+  searchMaillingLists();
```

Comment:
@matheus-fga , if I'm not wrong, this line was added so the lists can be rendered when the page has been rendered for the first time? So, after the ajax-generated page was merged, this test was crashing: https://github.com/colab/colab-superarchives-plugin/blob/master/tests/test_mailinglist_view.py#L158, so I basically removed this line and forced the  first time generation to be rendered by 'manage_subscriptions' (https://github.com/colab/colab/blob/detach_super_archives/colab/accounts/templates/accounts/manage_subscriptions.html) template, then the test was back working again